pkg/cmd/authTokens.go

```diff
@@ -76,3 +77,21 @@ func (cmd *AuthTokensCommand) createAuthTokensCobraCmd(
 
 	return authTokensCmd, err
 }
+
+func addLoginIdFlagToAuthTokensGet(cmd *cobra.Command, isMandatory bool, authTokensGetCmdValues *AuthTokensCmdValues) {
```

Comment:
Since this is specific to `auth tokens get`, it would make sense to put this function in `authTokensGet.go`